src/app/extensions/address-doctor/models/address-doctor-config.model.ts

```diff
@@ -0,0 +1,6 @@
+export interface AddressDoctorConfig {
+  login: string;
+  password: string;
```

Comment:
Hi,
I can't really find the recommendations from Address Doctor to use their apis, is it secure to store in the front the login and password to use their services ? Isn't there a risk for someone to collect it ?

Reply:
@charleshopfner The client needs to send the login and password with the POST REST request in plain text to the Address Doctor server. So the client will have to know these values at some point.
In a production environment these values are set via an environment variable at deployment time, so the values are not part of the actual source code and we assume that is "secret" enough.

But we are open to any suggestions how this could be handled better if necessary.